Integrations/Exabeam-new/Exabeam-new.py

```diff
@@ -0,0 +1,475 @@
+            'PeerGroupFieldName': user.get('peerGroupFieldName'),
+            'PeerGroupFieldValue': user.get('peerGroupFieldValue'),
+            'PeerGroupDisplayName': user.get('peerGroupDisplayName'),
+            'PeerGroupType': user.get('peerGroupType')
+        }
+
+    context = {
+        'Exabeam.User(val.UserName && val.UserName === obj.UserName)': contents
+    }
+
+    return_outputs(tableToMarkdown(f'User {username} information', contents, headers, removeNull=True), context, user)
+
+
+def watchlist_add_user(client, args):
+    """
+
+    Args:
+        client: Client
+
+    Add user to a watchlist
+
+    """
+
+    user_id = args.get('user_id')
+    watchlist_id = args.get('watchlist_id')
+
+    response = client.add_user_request(user_id, watchlist_id)
+    if response:
+        contents = {
+            'UserID': response.get('item'),
+            'WatchlistID': response.get('watchlistId')
+        }
+    context = {
+        'Exabeam.Watchlist(val.WatchlistID && val.WatchlistID === obj.WatchlistID)': contents
+    }
+
+    return_outputs(tableToMarkdown('The user was added successfully to the watchlist', contents), context, response)
+
+
+def get_watchlist(client):
+
+    watchlist = client.get_watchlist_request()
+    contents = []
+    headers = ['WatchlistID', 'Title', 'Category']
+    for list_ in watchlist:
+        contents.append({
+            'WatchlistID': list_.get('watchlistId'),
+            'Title': list_.get('title'),
+            'Category': list_.get('category')
+        })
+
+    context = {
+        'Exabeam.Watchlist(val.WatchlistID && val.WatchlistID === obj.WatchlistID)': contents
+    }
+
+    return_outputs(tableToMarkdown('Exabeam Watchlists', contents, headers), context, watchlist)
+
+
+def create_watchlist(client, args):
+    """
+    Create new watchlist
+    """
+    title = args.get('title')
+    category = args.get('category')
+    description = args.get('description')
+    items = argToList(args.get('items'))
+    headers = ['WatchlistID', 'Title', 'Category']
+
+    watchlist = client.create_watchlist_request(title, category, description, items)
+    if watchlist:
+        contents = {
+            'WatchlistID': watchlist.get('watchlistId'),
+            'Title': watchlist.get('title'),
+            'Category': watchlist.get('category')
+        }
+
+    context = {
+        'Exabeam.Watchlist(val.WatchlistID && val.WatchlistID === obj.WatchlistID)': contents
+    }
+    return_outputs(tableToMarkdown('New watchlist has been created', contents, headers), context, watchlist)
+
+
+def delete_watchlist(client: Client, args: Dict):
+    """
+    Delete a watchlist
+    """
+
+    watchlist_id = args.get('watchlist_id')
+    client.delete_watchlist_request(watchlist_id)
+
+    demisto.results('The watchlist was deleted successfully')
+
+
+def get_peer_groups(client):
+
+    groups = client.get_peergroups_request()
+    peer_group = ', '.join(groups)
+    new_group = peer_group.split(',')
```

Comment:
no need to join and then split, just split on `', '`

Reply:
Can't split on a list